pkg/agent/bakerapi_test.go

```diff
@@ -523,4 +526,22 @@ var _ = Describe("AgentBaker API implementation tests", func() {
 			}
 		})
 	})
+
+	Context("GetCachedVersionsOnVHD", func() {
+		It("should return cached VHD data", func() {
+			agentBaker, err := NewAgentBaker()
+			Expect(err).NotTo(HaveOccurred())
+
+			fromManifest, fromComponents, err := agentBaker.GetCachedVersionsOnVHD()
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(fromManifest["runc"].Installed["default"]).To(Equal("1.1.12"))
```

Comment:
yeah... updating this would be annoying but we do something similar already with sig_config sorta, so not the end of the world - best bet would be to manually load in those files during the test and compare with the current state of the values rather than hard-coding here